packages/create-atomic/script/npmInstallTemplate.ts

```diff
@@ -0,0 +1,24 @@
+import {spawn} from 'node:child_process';
+import {join} from 'node:path';
+import {cwd} from 'node:process';
+import {getPackageManager} from '../src/utils.js';
+
+(async () =>
+  await new Promise((resolve, reject) => {
+    const process = spawn(
+      getPackageManager(),
+      ['install', '--ignore-scripts'],
```

Comment:
The magic is in the `ignore-scripts`: it prevent the postinstall to run in the `template`, which prevent the deletion of its bits.